src/shared-components/optionButton/index.tsx

```diff
@@ -12,27 +12,48 @@ import {
 } from './style';
 import CheckmarkIcon from '../../svgs/icons/checkmark-icon.svg';
 
+export type OptionButtonProps = {
+  buttonType?: 'primary' | 'secondary';
+  icon?: React.ReactNode;
+  text: string;
+  subtext?: React.ReactNode;
+  onClick: () => void;
+  optionType: 'radio' | 'checkbox';
+  selected?: boolean;
+  [key: string]: unknown;
+};
+
 const OptionButton = ({
+  buttonType = 'primary',
   icon,
   text,
   onClick,
   optionType,
-  selected,
+  selected = false,
   subtext,
-  type,
   ...rest
-}) => (
-  <ClickableContainer onClick={onClick} type="clickable" tabIndex="0" {...rest}>
+}: OptionButtonProps) => (
+  <ClickableContainer
+    onClick={onClick}
+    type="button"
+    containerType="clickable"
```

Comment:
Unless I'm missing something, I think we also might want to pass `role={buttonType}`. We'll get the correct behavior from using a `button` either way, but I think these always either act as radio or checkbox toggles when they are clickable.

Reply:
Updated with `role={optionType}` (`buttonType` is `primary | secondary`, `optionType` is `checkbox | radio`).